router/cmd/boot/boot.go

```diff
@@ -80,6 +86,8 @@ func main() {
 	exitChan := make(chan os.Signal, 2)
 	signal.Notify(exitChan, syscall.SIGTERM, syscall.SIGINT)
 	<-exitChan
+	tail.Stop()
```

Comment:
The compiler failed this because `Cleanup` is a simple func but `Tail` must be called through an actual instance:
```go
func Cleanup() { ...

func (tail *Tail) Stop() error { ...
```